lib/restaurant.rb

```diff
@@ -0,0 +1,15 @@
+class Restaurant 
+  
+  attr_reader :menu
+  
+  def initialize(cuisine="Italian")
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `cuisine`. If it's necessary, use `_` or `_cuisine` as an argument name to indicate that it won't be used. You can also write as `initialize(*)` if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)